src/build_resume.py

```diff
@@ -49,13 +50,18 @@ def update_values(dict_values):
 
 
 def build_resume():
+    # switch to src directory
+    chdir(src_dir)
```

Comment:
this line is unnecessary if this is already being run from within the `src/` directory, and even then, does nothing since `src_dir` is defined as the current working directory anyway.